README.md

```diff
@@ -0,0 +1,69 @@
+# LAB - Class 32
+
+## Project: Django REST Framework Authentication & Production Server
+
+### Author: Katrina Hill
+
+- attribution to instructor JB Tellez for class demo code modified for this application
+
+### Setup
+
+- PORT - <http://0.0.0.0:8000/>
+- DATABASE_URL - <http://0.0.0.0:8000/api/v1/food/>
+
+### How to initialize/run the application
+
+#### Terminal 1
+
+- create virtual environment: python3 -venv .venv
+- activate virtual environment: source .venv/bin/activate
+- pip install pytest
+- pip freeze > requirements.txt
+- Add appropriate/applicable installed apps, allowed hosts, rest framework, static urls, admin, permissions, views, urls, and database default in respective files.
+
+#### Terminal 2
+
+- docker compose up
+- docker-compose run web bash
+- pip install django
```

Comment:
`docker compose up` will build/update the container image then run the container. When it builds/updates the container image, this specific step is being done by the Dockerfile. Inside that Dockerfile it's loading all the apps in requirements.txt. A pip freeze has already been done by the creator of this code (you!), so the requirements.txt file is already ready to roll (looks good!: [https://github.com/katrina-l-hill/drf-auth/blob/main/requirements.txt](https://github.com/katrina-l-hill/drf-auth/blob/main/requirements.txt)). When `docker compose up` happens, it looks at requirements.txt, and creates an image with all of these packages baked into it: django, djangorestframework, pscopg2-binary, gunicorn, etc etc.